muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/workspaces/index.ts

```diff
@@ -357,7 +359,7 @@ export interface OrganizationWorkspacesType {
   activeFilters: OrganizationWorkspacesActiveFiltersType,
   hasMore: boolean,
   toolbarLock: boolean,
-  types?: Array<WorkspaceTypeType>
+  types?: Array<WorkspaceTypeType>,
```

Comment:
Why comma after last element?

Reply:
Trailing commas in type/array/object literals are more of a styling preference. We might as well decide in general whether we use them or not. I took habit of adding them which is or is not the preferred way.

We could even add rule about trialing commas to the hound's typescript linter and fix them one by one eventually.